### PR TITLE
Application details ginis data

### PR DESCRIPTION
**Pull request by mpinter**
Details about document owner + stub for GINIS document history.

Edit 2: what's still missing
- fix data passed into history - today we'll display 2 'document created' history states and nothing more. Waiting for feedback on how to map GINIS history states.

### CODE REVIEW

File: next/components/forms/segments/AccountSections/MyApplicationsSection/MyApplicationHistory.tsx [hunk 1]
```diff
@@ -11,20 +13,32 @@ const MyApplicationHistory = ({ historyData }: MyApplicationHistoryProps) => {
     <>
       {/* Desktop */}
       <div className="hidden w-full flex-col gap-4 md:flex">
-        {historyData?.map((data, i) => (
+        {/* TODO - history data needs changes in process and on BE - until then, we just take 1 instance and present it as 'document created' (all the instances are interpreted as document created) */}
+        {historyData?.slice(-1).map((data: any, i) => (
```
Comment (MarekBodingerBA) [inline next/components/forms/segments/AccountSections/MyApplicationsSection/MyApplicationHistory.tsx:17]: Is it `any` on purpose?

File: next/components/forms/segments/AccountSections/MyApplicationsSection/MyApplicationHistory.tsx [hunk 2]
```diff
@@ -42,10 +56,21 @@ const MyApplicationHistory = ({ historyData }: MyApplicationHistoryProps) => {
...
           <div className="flex flex-col items-center">
-            {historyData?.map((data, i) => (
+            {historyData?.map((data: any, i) => (
               <div className="flex w-full border-t-2 p-4" key={i}>
-                <span className="text-p2 min-w-[240px]">{data?.editDate}</span>
-                <span className="text-p2 w-full">{data?.description}</span>
+                <span className="text-p2 min-w-[240px]">
+                  <FormatDate>
+                    {/* eslint-disable-next-line @typescript-eslint/no-unsafe-member-access */}
+                    {data?.DatumZmeny}
```
Comment (MarekBodingerBA) [inline next/components/forms/segments/AccountSections/MyApplicationsSection/MyApplicationHistory.tsx:64]: Check if FormatDate doesn't fail when the date is undefined / null.

File: next/components/forms/segments/AccountSections/MyApplicationsSection/MyApplicationHistory.tsx (hunk 1 above)
Comment (mpinter) [reply next/components/forms/segments/AccountSections/MyApplicationsSection/MyApplicationHistory.tsx:17]: yes, that's the TODO on top `fix the types in OpenAPI (BE)` - will add a comment here as well

File: next/components/forms/segments/AccountSections/MyApplicationsSection/MyApplicationHistory.tsx (hunk 2 above)
Comment (mpinter) [reply next/components/forms/segments/AccountSections/MyApplicationsSection/MyApplicationHistory.tsx:64]: it doesn't, it'll print "Invalid Date"

**Pull request merged**